specs/rest-api.md

```diff
@@ -699,9 +699,11 @@ In the source container case, the following dict must be passed:
 
     {
         "public": true,             # True or False
+        "name":   imagename,        # Name for new image
```

Comment:
^ what's that?

Our images don't have names, they just have the hash. The description for the image is the "description" property.